descriptors/ModuleDescriptor-template.json

```diff
@@ -22,7 +22,7 @@
     },
     {
       "id": "inventory-reports",
-      "version": "0.2",
+      "version": "0.3",
```

Comment:
I think we have to update the version in RAML as well.

Reply:
Updated